app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def user; end
+
+  def posts; end
+
+  def post; end
+end
```

Comment:
- [ ] Apart from this users_controller, let's also add `posts_controller` with `index` and `show` actions as requested in the project requirement.